frontend/src/components/common/table/table.tsx

```diff
@@ -2,59 +2,80 @@ import { useTable } from 'hooks/hooks';
 import { Column, useSortBy } from 'react-table';
 import styles from './styles.module.scss';
 import { getAllowedClasses } from 'helpers/helpers';
+import { Pagination } from './components/components';
 
 type Props = {
   columns: Column[];
   data: unknown[];
+  handleSetRow: (row: number) => void;
+  handleSetPage: (page: number) => void;
+  pageIndex: number;
+  pageSize: number;
+  totalCountEpisodes: number;
```

Comment:
check **A8** criteria
add new props `hasPagination` and make all these props optionals